### ISSUE TITLE
Last custom field is not saved

### ISSUE THREAD

**Issue by GoogleCodeExporter** [viktorthang/wordpress-custom-content-type-manager#171]
```
What steps will reproduce the problem? (Be specific)
1. Edit a custom content type with multiple rich text fields.
2. Enter text in the last field.
3. Click Publish
4. Observe that no content was saved for last field.

(see attached screencast for details)

What is the expected output?
Saved content for all fields.

What do you see instead?
No content in final field.

Does the problem continue if you disable all other plugins?
Yes.

Please provide any additional information below (e.g. Browser, version,
OS):
Firefox 3.6, 6.0, Chrome 13

Please copy and paste the system information from the CCTM WP dashboard
screen that appears in a yellow textarea (this includes the version of the
plugin, the version of PHP, the version of MySQL, a list of other active
plugins etc.):
*SYSTEM INFO* 
------------------------ 
Plugin Version: 0.9.4 dev
WordPress Version: 3.2.1
PHP Version: 5.2.17
MySQL Version: 5.1.53-log
Server OS: Linux
------------------------ 
Other Active plugins: 
 * Advanced Menu Widget v.0.1 [http://www.techforum.sk/]
 * Bulletins Widget v.1.0 [http://calnewman.org]
 * Page in Widget v.1.2 [http://carl-fredrik.net/wordpress/page-in-widget.html]

```

Original issue reported on code.google.com by `ryangree...@gmail.com` on 10 Sep 2011 at 6:40

Attachments:
- [last_custom_field_not_saved.mov](https://storage.googleapis.com/google-code-attachments/wordpress-custom-content-type-manager/issue-171/comment-0/last_custom_field_not_saved.mov)

**Comment by GoogleCodeExporter** [comment]
```
Thanks for the detailed report!  I'll be looking into this shortly.
```

Original comment by `fireproofsocks` on 10 Sep 2011 at 6:52
- Changed state: **InReview**

**Comment by GoogleCodeExporter** [comment]
```
Verified.  Man, that one is weird.
```

Original comment by `fireproofsocks` on 16 Sep 2011 at 3:57
- Changed state: **InProgress**

**Comment by GoogleCodeExporter** [comment]
```
Wow... this is a pain.  Check out this post: 
http://wordpress.org/support/topic/how-i-can-use-tinymce-for-my-own-plugin?repli
es=8

It should work "as long as you make sure that they have the same class and id, 
but different names."    Sort of... here's the fix:

Edit includes/elements/wysiwyg.php line 119 from this:

    , $this->get_field_class($this->name, 'text') . ' ' . $this->class

To this:

    , 'cctm_wysiwyg'

This has been fixed and will in version 0.9.4
```

Original comment by `fireproofsocks` on 16 Sep 2011 at 4:45
- Changed state: **Fixed**
